tests/debugger/test_debugger.py

```diff
@@ -194,6 +194,7 @@ def setup_method_probe_snaphots(self):
         self.span_probe_response = weblog.get("/debugger/span")
         self.span_decoration_probe_response = weblog.get("/debugger/span-decoration/asd/1")
 
+    @missing_feature(context.library >= "java@1.27", reason="introduction of new EMITTING probe status")
```

Comment:
But with the latest changes we aren't testing the "EMITTING" status....
I guess it should be something  similar to this:

```
def method_probe_snaphots(self,  expected_probes):
    ... assertions....
@irrelevant(context.library >= "java@1.27", reason="introduction of new EMITTING probe status")
def test_method_installed_probe_snaphots(self):
        expected_probes = {
            "log170aa-acda-4453-9111-1478a6method": "INSTALLED",
            "metricaa-acda-4453-9111-1478a6method": "INSTALLED",
            "span70aa-acda-4453-9111-1478a6method": "INSTALLED",
            "decor0aa-acda-4453-9111-1478a6method": "INSTALLED",
        }
       self.method_probe_snaphots(expected_probes)

@missing_feature(context.library < "java@1.27", reason="last releases use new EMITTING probe status")
def test_method_emitting_probe_snaphots(self):
        expected_probes = {
            "log170aa-acda-4453-9111-1478a6method": "EMITTING",
            "metricaa-acda-4453-9111-1478a6method": "EMITTING",
            "span70aa-acda-4453-9111-1478a6method": "EMITTING",
            "decor0aa-acda-4453-9111-1478a6method": "EMITTING",
        }
       self.method_probe_snaphots(expected_probes)
```